dummy-data/src/index.ts

```diff
@@ -80,7 +80,7 @@ async function installComponent(
     octokit,
     owner: 'exivity',
     repo: component,
-    ref: 'master',
+    ref: STANDARD_BRANCH,
```

Comment:
@CalliEve - All components now only have a main branch (apart from scaffold), so I renamed this, correct?

Reply:
yeah should be fine!